ESO/README.md

```diff
@@ -119,8 +119,11 @@ Runs the script without executing the simulations to check input values.
 runs the script with automatically generated calibration files, and writes the calibration files to a
 recipe YAML file. 
 
-runs the script with 32x32 pixel outputs, for testing purposes. 
```

Comment:
This line about 32x32 pixel outputs belonged to the `--small` parameter. It's seems to go lost in some reorganization.